src/main/java/core/basesyntax/service/impl/FileDataParserImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.FileDataParser;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileDataParserImpl implements FileDataParser {
+    @Override
+    public List<Transaction> parse(List<String> data) {
+        return data.stream()
+                .skip(1)
+                .map(this::parseTransaction)
+                .collect(Collectors.toList());
+    }
+
+    private Transaction parseTransaction(String data) {
+        String[] dataArray = data.split(",");
```

Comment:
I think it's better to use a constant instead of ",".

Reply:
🥲